styles.css

```diff
@@ -5,3 +5,50 @@
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --salmon-color: #f0aea9;
+  --light-skyblue-color: #8bd9f6;
+  --light-brown-color: #f8b172;
+  --light-green-color: #93cb5b;
+  --dark-green-color: #6bca6b;
+  --light-orange-color: #f1ba33;
+  --dark-orange-color: #f9aa23;
+  --coral-color: #f28373;
```

Comment:
Now that I look at it, it is probably fine to drop color from the variable names. 

We could just have `light-green`, `light-orange` , etc.